lib/ClangImporter/SwiftLookupTable.h

```diff
@@ -133,8 +133,14 @@ class EffectiveClangContext {
     return StringRef(Unresolved.Data, UnresolvedLength);
   }
 };
+
+#if __SIZEOF_POINTER__ == 4
```

Comment:
Nope. __SIZEOF_POINTER__ is undefined 

Use LLVM_PTR_SIZE defined in "LLVM/Support/Compiler.h"

https://github.com/llvm-mirror/llvm/blob/master/include/llvm/Support/Compiler.h